docs/source/test_network.md

```diff
@@ -501,7 +501,7 @@ The command will reveal the MSP folder structure and configuration file:
 organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/
 └── msp
     ├── IssuerPublicKey
-    ├── IssuerRevocationPublicKey
+    ├── RevocationPublicKey
```

Comment:
The right thing to do here might be to change idemix itself to match the CA. I opened #1557 lets see what the Ale and Artem say on this.

Reply:
Great, instead of maintaining two names as confusion handler, Code change could be better!